lib/wprof/conf_wprof.rb

```diff
@@ -0,0 +1,26 @@
+module WProf
+  class Config
+    DEFAULTS_CONFIGS = {
+      db_runtime: true,
+      reporter_type: 'LOGGER',
+      csv_type: 'SPLIT',
+      async: false,
+      httparty_methods_to_trace: %w[get put delete post].freeze,
+      external_url: nil,
+      external_headers: nil,
+      custom_methods: nil,
+      file_path: Rails.root.to_s
+    }.freeze
+
+    class << self
+      def get_value(param)
+        unless Rails.application.nil?
+          Rails.configuration.x.wprof.each do |key, value|
```

Comment:
Can we get the value with something like:
```
Rails.configuration.x.wprof[param]
```
Instead of looping over the hash

Reply:
Yes!!!!!!!!!!!!!!!!!!!! Of Course!! I don't know why i made that this! Changed!